ontology/hso.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: ontology_detail
+id: genepio
+contact:
+  email: fernanda.dorea@sva.se
+  label: Fernanda Dorea
+description: The health Surveillance Ontology (HSO) focuses on "surveillance system level data", that is, data outputs from surveillance activities, such as number of samples collected, cases observed, etc. It aims to support One-Health surveillance, covering animal health, public health and food safety surveillance.  
+domain: health
+homepage: https://w3id.org/hso
+page: https://github.com/SVA-SE/HSO
+license:
+  url: http://creativecommons.org/licenses/by/3.0/
+  label: CC-BY
+products:
+  - id: hso.owl
+    homepage: https://w3id.org/hso
+title: Health Surveillance Ontology
+dependencies:
+ - id: uberon
+ - id: bfo
+ - id: ro
+ - id: obi
+ - id: ncbitaxon
+tracker: https://github.com/SVA-SE/HSO/issues/
+
+activity_status: active
+---
+
+The health Surveillance Ontology (HSO) focuses on "surveillance system level data", that is, data outputs from surveillance activities, such as number of samples collected, cases observed, etc. It aims to support One-Health surveillance, covering animal health, public health and food safety surveillance.
```

Comment:
Maybe it would be good to give an example of what health surveillance is and also what the term One-health surveillance means.. Its helpful if the description gives at least a sense of what kind of terms the ontology contains.